### ISSUE TITLE
Deleted messages don't purge attachments

### ISSUE THREAD

**Issue by Hildy** [jstanden/cerb#459]
When you delete a message from a conversation, the attachments it had are then orphaned (and not deleted), even after maint runs.

**Comment by Hildy** [comment]
Similar issue to #458

**Comment by iammattmartin** [comment]
Is there way to manually purge attachments which no longer have their ticket/message entity whilst this is being worked on? Doing them one by one is quite tedious.

**Comment by jstanden** [comment]
Fixed in 8.2.6

**Comment by jstanden** [comment]
@mattmartincouk 

In the database:
~~~
DELETE FROM attachment_link WHERE context = 'cerberusweb.contexts.message' AND context_id NOT IN (SELECT id FROM message);

DELETE FROM attachment_link WHERE context = 'cerberusweb.contexts.comment' AND context_id NOT IN (SELECT id FROM comment);
~~~

Then nightly maintenance will properly remove attachments with no links left.
